conf/nginx.conf

```diff
@@ -1,22 +1,34 @@
-location ^~ __PATH__ {
+#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
+location __PATH__/ {
+
+  # Path to source
   alias __FINALPATH__/public/;
-try_files $uri $uri/ @kimai;  
-index index.php index.htm index.html;
-    # Force https
+  
+  # Force usage of https
   if ($scheme = http) {
     rewrite ^ https://$server_name$request_uri? permanent;
   }
+
+  index index.php index.htm index.html;
+
+  # Common parameter to increase upload size limit in conjunction with dedicated php-fpm file
+  #client_max_body_size 50M;
+
+  try_files $uri $uri/ @kimai;
```

Comment:
https://www.kimai.org/documentation/webserver-configuration.html#nginx

Did you try if request parameters work? I had that problem often in the past that searches / pagination did not work because they were not passed to PHP.
The rule I recommend is
```
        try_files $uri /index.php$is_args$args;
```

Reply:
switched to `try_files $uri /index.php$is_args$args;`